PowerEditor/installer/nativeLang/chineseSimplified.xml

```diff
@@ -12,6 +12,7 @@
                     <Item menuId="encoding" name="格式(&amp;M)" />
                     <Item menuId="language" name="语言(&amp;L)" />
                     <Item menuId="settings" name="设置(&amp;T)" />
+                    <Item menuId="tools" name="工具(&amp;O)"/>
```

Comment:
Problem: there are 2 `&amp O`
Could you please aligned to chinese.xml, please?
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/chinese.xml